test/tests/TestData.h

```diff
@@ -13,6 +13,6 @@
 
 namespace TestData
 {
-    std::string GetBasePath();
-    std::string GetParkPath(std::string name);
-};
+std::string GetBasePath();
```

Comment:
Not indenting without a namespace does not look nice.

Reply:
I'm personally partial to not indenting namespaces, especially with nesting them having become unnecessary as of C++17.

Reply:
I don't think its possible to configure.

Reply:
`NamespaceIndentation` could be set to `Inner` and `All`. I've brought this up before [here](https://github.com/OpenRCT2/OpenRCT2/pull/7178#discussion_r168868261). So your preference is to indent all namespaces, Aaron's is to not indent at all, and mine is to indent only inner namespaces.